### ISSUE TITLE
Capitalized extensions

### ISSUE THREAD

**Issue by JanBorowskiES** [prine/ROThumbnailGenerator#1]
Hello. Cool library, but I believe there is one problem with it. My app crashed when I wanted to get thumbnail of file with name `IMG_0007.JPG`. I think that the problem is uppercase file extension, because when I changed `ROThumbnail.sharedInstance.getThumbnail` method with following code it worked:

```
    public func getThumbnail(url:NSURL) -> UIImage {
        if let fileExtension = url.pathExtension {
            var smallExt = fileExtension.lowercaseString
            var appropriateThumbnailGenerator = supportedFiletypes[smallExt] ?? DefaultThumbnailGenerator()
            return appropriateThumbnailGenerator.getThumbnail(url)
        }

        return UIImage(named:"fallbackIcon")!
    }
```

**Comment by prine** [comment]
Ah good catch! Will fix it in the next release. Thanks!

**Comment by prine** [comment]
Fixed in Commit a9de6a889c04cc6ff257b32afc7555f993de0f17
